### PR TITLE
[Workers] remove `publish` & replace with `deploy`

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-7755

`publish` is deprecated in Wrangler v3.

### CODE REVIEW

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/workers/wrangler/ci-cd.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To create a token:
 CLOUDFLARE_API_TOKEN=<YOUR_API_TOKEN_VALUE>
 ```
 
-You can also inline the value directly with the `wrangler publish` command. For example: `CLOUDFLARE_API_TOKEN=XXX wrangler publish`.
+You can also inline the value directly with the `wrangler deploy` command. For example: `CLOUDFLARE_API_TOKEN=XXX wrangler deploy`.
```
Comment (deadlypants1973) [inline content/workers/wrangler/ci-cd.md:28]: devprod

File: content/workers/wrangler/system-environment-variables.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ System Environment Variables are local environment variables that can change Wra
 
 1. Create an `.env` file in your project directory. Set the values of your environment variables in your [`.env`](/workers/wrangler/system-environment-variables/#example-env-file) file. This is the recommended way to set these variables, as it persists the values between Wrangler sessions.
 
-2. Inline the values in your Wrangler command. For example, `WRANGLER_LOG="debug" wrangler publish` will set the value of `WRANGLER_LOG` to `"debug"` for this execution of the command.
+2. Inline the values in your Wrangler command. For example, `WRANGLER_LOG="debug" wrangler deploy` will set the value of `WRANGLER_LOG` to `"debug"` for this execution of the command.
```
Comment (deadlypants1973) [inline content/workers/wrangler/system-environment-variables.md:12]: devprod

Comment (admah) [reply content/workers/wrangler/system-environment-variables.md:12]: 👍

**Review by admah — APPROVED** [review]
This looks great. Thanks!

**Pull request merged**